cypress/utils/cart.ts

```diff
@@ -0,0 +1,33 @@
+/*
+  Pages: *
+  Selectors: openCart
+*/
+export function openCart() {
+  cy.get('[data-testid="openCart"').click()
+}
+
+/*
+  Pages: *
+  Selectors: openCart
```

Comment:
Did you mean?
```suggestion
  Selectors: closeCart
```

Reply:
Thanks! I'm writing a doc right now, I will try to add these examples there!!